src/pal/src/cruntime/math.cpp

```diff
@@ -453,6 +453,34 @@ PALIMPORT double __cdecl PAL_pow(double x, double y)
     return ret;
 }
 
+/*++
+Function:
+  PAL_signbit
+
+Determines whether whether the sign of the double-precision floating point value is negative.
+
+Return Value
+
+PAL_signbit returns a nonzero value (TRUE) if the sign of its argument x is  negative.
+
+Parameter
+
+x  Double-precision floating-point value
+
+--*/
+int __cdecl PAL_signbit(double x)
```

Comment:
Had to add `PAL_signbit`.... Clang complains about importing `fp.h` and using `FPDOUBLE` to get the sign-bit, because the header also declares a static function which is unused in `grisu3.cpp`

Rather than fix the header or compilation flags, I opted to just use `signbit`, since it is the standard library function to do this.

Reply:
Adding it to `palinternal.h` doesn't work in this case, because it is a macro in the `math.h` file.

I had to do what the other "macro" PAL redefinitions are doing (like `isfinite` and `isnan`), which is to define a new function (`_signbit`) which calls the macro, and then use the new function everywhere.
For the existing ones, the new function name was selected to match the MSVC function name (`_finite`, `_isnan`, etc). Since there is no `_signbit` function for MSVC (just `signbit`), I had to (for MSVC) also do `#define signbit _signbit`.